blog: Introduction to UNS (#1413)
FlowFuse/website: src/blog/2023/12/introduction-to-unified-namespace.md

---
title: "Introduction to the Unified Namespace"

> subtitle: ""
> description: ""

Author: ```suggestion
subtitle: "Making data available for Industry 4.0 use-cases"
description: "Making data available for Industry 4.0 use-cases"
```

authors: ["zeger-jan-van-de-weg"]
date: 2023-12-20
image: "/blog/2023/11/images/raspberry-pi-5-device-agent.png" #todo
tags:
    - posts
    - flowfuse
---

As your organization is generating more data there’s key architectural decisions to be made to ensure the full value can be unlocked and you’re leveraging not just the tip of the iceberg. The Unified Namespace provides a blueprint to allow data to be consumed by many data-consumers.

> 

Reviewer: ```suggestion
<!--more-->
```

> 

Reviewer 2: ```suggestion

<!--more-->

```

Reviewer 2: Adding for visualization

> To facilitate a many to many connection between data producers and data consumer, there are two changes to be made for your architecture:

Reviewer 2: ```suggestion
To facilitate a many to many connection between data producers and data consumers, there are two changes to be made to your architecture:
```

> As your organization is generating more data there’s key architectural decisions to be made to ensure the full value can be unlocked and you’re leveraging not just the tip of the iceberg. The Unified Namespace provides a blueprint to allow data to be consumed by many data-consumers.
> <!--more-->
> To facilitate a many to many connection between data producers and data consumer, there are two changes to be made for your architecture:

Author: ```suggestion
As your organization is generating more data there’s key architectural decisions to be made to ensure the full value can be unlocked and you’re leveraging not just the tip of the iceberg. The Unified Namespace provides a blueprint to allow data to be consumed by many data-consumers.

<!--more-->

To facilitate a many to many connection between data producers and data consumer, there are two changes to be made for your architecture:
```

Traditionally, for example web servers serving web pages, the client requests a
page from a server. This is a point to point connection between those two parties.

> ![Point to point graphic](./images/uns-point-to-point.png)

Author: ```suggestion
![Point to point graphic](./images/uns-point-to-point.png "Point to Point connection")
```
